tests/trainer/flags/test_min_max_steps_epochs.py

```diff
@@ -0,0 +1,41 @@
+from tests.base.boring_model import BoringModel
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import EarlyStopping
+
+
+def test_max_steps_only(tmpdir):
+    """
+    Tests that max_steps can be used without max_epochs
+    """
+    model = BoringModel()
+
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        min_epochs=0,
+        max_steps=3,
+        min_steps=0,
+        weights_summary=None,
+    )
```

Comment:
could you also test with a larger max_steps value? i want to test a case where `trainer.current_epoch` should be > 1 even if `max_epochs` is `None`